docs/training/apis-howto/api-core-ug.rst

```diff
@@ -1,212 +1,302 @@
+| :ref:`core-reference`                                            |
 +------------------------------------------------------------------+
 
-With the Core API you can train arbitrary models on the Determined platform with seamless access to
-the the following capabilities:
+In this user guide, we'll show you how to adapt model training code to use the Core API. As an
+example, we'll be working with the PyTorch MNIST model.
 
--  metrics tracking
--  checkpoint tracking and preemption support
--  hyperparameter search
--  distributing work across multiple GPUs and/or nodes
+************
+ Objectives
+************
 
-These are the same features provided by the higher-level PyTorchTrial, DeepSpeedTrial, and
-TFKerasTrial APIs (the higher-level APIs are implemented using the Core API).
+These step-by-step instructions walk you through modifying a script for the purpose of performing
+the following functions:
 
-This user guide shows you how to get started using the Core API.
+-  Report metrics
+-  Report checkpoints
+-  Perform a hyperparameter search
+-  Perform distributed training
 
-.. _core-getting-started:
+After completing the steps in this user guide, you will be able to:
 
-*****************
- Getting Started
-*****************
+-  Understand the minimum requirements for running an experiment
+-  Modify a script and an experiment configuration file
+-  Understand how to convert model code
+-  Use the Core API to train a model
 
-As a simple introduction, this example training script increments a single integer in a loop,
-instead of training a model with machine learning. The changes shown for the example model should be
-similar to the changes you make in your actual model.
+***************
+ Prerequisites
+***************
 
-The ``0_start.py`` training script used in this example contains your simple "model":
+**Required**
 
-.. literalinclude:: ../../../examples/tutorials/core_api/0_start.py
-   :language: python
-   :start-at: import
-
-To run this script, create a configuration file with at least the following values:
+-  A Determined cluster
 
-.. literalinclude:: ../../../examples/tutorials/core_api/0_start.yaml
-   :language: yaml
+**Recommended**
 
-The actual configuration file can have any name, but this example uses ``0_start.yaml``.
+-  :ref:`qs-mdldev`
 
-Run the code using the command:
+*****************************************************
+ Step 1: Get the Tutorial Files & Run the Experiment
+*****************************************************
 
-.. code:: bash
+To run an experiment, you need, at minimum, a script and an experiment configuration (YAML) file.
 
-   det e create 0_start.yaml . -f
+Create a new directory.
 
-If you navigate to this experiment in the WebUI no metrics are displayed because you have not yet
-reported them to the master using the Core API.
+Get the files from the :download:`core_api_pytorch_mnist.tgz </examples/core_api_pytorch_mnist.tgz>`
```

Comment:
"Get the files by downloading and extracting this link" or similar

Reply:
resolve. the new content now states:

Retrieve the files from the :download:`core_api_pytorch_mnist.tgz
</examples/core_api_pytorch_mnist.tgz>` download or download from the `Github repository
<https://github.com/determined-ai/determined/tree/master/examples/tutorials/core_api_pytorch_mnist>`_.
These scripts have already been modified to fit the steps outlined in this tutorial.